tests/test_wind_turbine.py

```diff
@@ -97,3 +97,13 @@ def test_wrongly_defined_to_group_method(self):
                            match="The 'number' and the 'total_capacity' "
                                  "parameter are mutually exclusive."):
             e_t_1.to_group(5, 3000)
+
+    def test_create_unphysical_turbine(self):
+        char = {
+           'hub_height': 80,
+           'rotor_diameter': 160,
+           'turbine_type': 'DUMMY 3',
+           'path': self.source
+        }
+        with pytest.raises(ValueError):
+            WindTurbine(**char)
```

Comment:
I would use the match attribute to make sure the right ValueError is raised.

```python
 with pytest.raises(ValueError, match=regex)
```

Reply:
Implemented. However, I had a problem with match when there was an asterisk symbol in the string so I modified the error message slightly.